edweiss-app/components/memento/CardScreenComponent.tsx

```diff
@@ -86,69 +115,38 @@ const CardScreenComponent: ReactComponent<{ deckId: string, cardIndex: number; i
         );
     };
 
-    // Animated styles for front and back of the card
-    const fronCardStyle = useAnimatedStyle(() => {
-        return {
-            transform: [{ perspective: 1000 }, { rotateY: `${rotation.value}deg` }]
-        };
-    });
-
-    const backCardStyle = useAnimatedStyle(() => {
-        return {
-            transform: [{ perspective: 1000 }, { rotateY: `${rotation.value + 180}deg` }]
-        };
-    });
-
     // Delete card
-    async function deleteCard() {
-
-        const res = await callFunction(Memento.Functions.deleteCards, { deckId: deckId, cardIndices: [cardIndex] });
-        if (res.status == 1) {
-            console.log(`Card deleted with id ${res.data.id}`);
-            isModal ? modalRef.current?.dismiss() : router.back();
-        }
-
+    const deleteCard = async () => {
```

Comment:
Why did you transform a function into this lambda?

Reply:
So that I can pass this function to a component that requires a function as its mandatory argument.

Reply:
Hmm I believe you can still do this with a regular function. (In fact, I use this a lot)

Reply:
You are right but since this implementation still works well so I will keep in mind your suggestion for future modification.